lib/wibox/widget/imagebox.lua

```diff
@@ -79,36 +75,38 @@ end
 --- Set an imagebox' image
 -- @property image
 -- @param image Either a string or a cairo image surface. A string is
---   interpreted as the path to a png image file.
+--   interpreted as the path to a image file.
 -- @return true on success, false if the image cannot be used
-
 function imagebox:set_image(image)
     if type(image) == "string" then
-        image = surface.load(image)
-        if not image then
-            print(debug.traceback())
-            return false
+        -- try to load rsvg handle from file if it's vector
+        local is_vector = string.match(image, "%.svg$")
```

Comment:
Please use `:match()`, this is used more often in the codebase (but not by much...).